edenia/antelope-tools
# Feat/enable evm for eos 1353

### Enable EVM Dashboard for EOS and Jungle4

### What does this PR do?

- Resolve #1353 
- Update the last block num with every request
- Total wallets created calculation is pending

### Steps to test

1. Run the project locally with EOS configuration
2. Go to EVM Dashboard Page
3. Check the dashboard

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/331ccb4d-faf6-4e97-acdd-b1a3facc49df)
